client/src/components/ChatBox.vue

```diff
@@ -1,37 +1,50 @@
 <template>
-    <form class="chatbox" @submit.prevent="sendMessage()">
-      <input v-model="message" class="chatbox__input" placeholder="Send a Message" />
-    </form>
+  <form class="chatbox" @submit.prevent="sendMessage()">
+    <input v-model="message" class="chatbox__input" placeholder="Send a Message" />
+    <button type="submit" class="like" name="foo" value="bar">
+      <svg
+        xmlns="http://www.w3.org/2000/svg"
+        width="24"
+        height="24"
+        viewBox="0 0 24 24"
+        fill="none"
+        stroke="currentColor"
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
+        class="feather feather-send"
+      >
+        <line x1="22" y1="2" x2="11" y2="13" />
+        <polygon points="22 2 15 22 11 13 2 9 22 2" />
+      </svg>
+    </button>
+  </form>
 </template>
 
 <script>
-import { ref } from '@vue/composition-api';
-import { useState, useActions } from '@u3u/vue-hooks';
+import { ref } from "@vue/composition-api";
```

Comment:
Use single quotes instead of double quotes.

Reply:
Good spot thank you